trailblazer/services/analysis_service/utils.py

```diff
@@ -47,7 +47,7 @@ def create_update_analyses_response(analyses: list[Analysis]) -> UpdateAnalysesR
 
 def get_upload_date(analysis: Analysis) -> datetime | None:
     completed_job: Job | None = _get_completed_job(analysis.upload_jobs)
```

Comment:
Not for this PR to solve, but I it is slightly annoying that analysis.upload_jobs is typehinted as singular here. Is it plural or singular?

Reply:
It is plural, but the function just retrieves one of the completed upload jobs.

Usually there is only one upload job, but there can also be several failed ones for FASTQ